test/test.js

```diff
@@ -58,14 +57,6 @@ if (process.platform === 'win32') {
 	});
 }
 
-const testNullOptions = async (t, execaMethod) => {
-	const {stdout} = await execaMethod('noop.js', [foobarString], null);
-	t.is(stdout, foobarString);
-};
-
-test('Can pass null to options', testNullOptions, execa);
-test('Can pass null to options - sync', testNullOptions, execaSync);
```

Comment:
This test was documenting that `execa('file', [...args], null)` currently does not throw.
This PR breaks this behavior. We could still support passing `null`, but it feels like this is not something we should support, and it is undocumented.

Reply:
Yeah, it should not be supported.